alts/src/main/java/io/grpc/alts/CheckGcpEnvironment.java

```diff
@@ -63,13 +63,15 @@ static boolean checkBiosDataOnWindows(BufferedReader reader) throws IOException
     }
     return false;
   }
+  
+  private static final String OS = System.getProperty("os.name").toLowerCase(Locale.ENGLISH);
```

Comment:
Don't make this a static variable; move it into isRunningOnGcp. It is only used in isRunningOnGcp and isRunningOnGcp is only called once.